packages/MarkdownEditor-Tests.package/MarkdownEditorTests.class/instance/testApplyingThemeDoesNotThrowError.st

```diff
@@ -0,0 +1,10 @@
+tests
+testApplyingThemeDoesNotThrowError
+
+	| textMorph |
+	textMorph := PluggableTextMorphPlus new
```

Comment:
You have a `toolbuilder` to build a `TextMorph` like it is in our application:
https://github.com/hpi-swa-teaching/MarkdownEditor/blob/e1053bf3baa21b38b6fcbf1a045f40704b388d99/packages/MarkdownEditor-Tests.package/MarkdownEditorTests.class/instance/testTextPersistence.st#L5